index.html

```diff
@@ -363,10 +361,9 @@ <h4 class="media-heading">No More Operations</h4>
 	    <div class="media-body">
 	      <h4 class="media-heading">Make Money</h4>
 	      <p>
-		Sell your application on our app store,
-		or sell virtual goods with in-app purchases.
-		Instant deployment at scale makes it trivial for
-		newcomers to challenge goliaths in the industry.
+		Take donations, show advertisements,
+		or charge for real world services.
+    There are plenty of business models that don't betray user trust.

```

Comment:
we're not going to do DRM - so we're not promising to stop piracy, especially if there's a binary a user controls that they can redistribute easily. While these models can thrive because most users are honest, I wonder if that's the business model (pay for features) that we want to be promoting.